test/e2e-travis.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# shellcheck disable=SC1090
+
+set -Eeuo pipefail
+
+echo '>> Deploy latest OLM'
+olm_version=$(curl -sL \
+  https://api.github.com/repos/operator-framework/operator-lifecycle-manager/releases/latest \
+  | grep -oP '"tag_name": "\K(.*)(?=")')
+curl -L https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/install.sh \
+  | bash -s "$olm_version"
```

Comment:
@cardil do we really need OLM? There's nothing in this upstream operator that uses OLM specific things ATM.

Reply:
You're right. It will fly without it. I assumed that it will be needed, but it isn't used as `ko` deploys operator.

I'll remove it.